dags/bqetl_kpi_dashboard.py

```diff
@@ -0,0 +1,54 @@
+# Generated via query_scheduling/generate_airflow_dags 
+
+from airflow import DAG
+from airflow.operators.sensors import ExternalTaskSensor
+import datetime
+from utils.gcp import bigquery_etl_query
+
+default_args = {'owner': 'jklukas@mozilla.com', 'email': ['telemetry-alerts@mozilla.com', 'jklukas@mozilla.com'], 'depends_on_past': False, 'start_date': datetime.datetime(2020, 5, 12, 0, 0), 'retry_delay': 'datetime.timedelta(seconds=600)', 'email_on_failure': True, 'email_on_retry': True, 'retries': 1}
+
+with DAG('bqetl_kpi_dashboard', default_args=default_args, schedule_interval=datetime.timedelta(0)) as dag:
+
+    telemetry_derived__smoot_usage_new_profiles__v2 = bigquery_etl_query(
+        destination_table='smoot_usage_new_profiles',
+        dataset_id='telemetry_derived',
+        project_id='moz-fx-data-shared-prod',
+        owner='jklukas@mozilla.com',
+        email=['jklukas@mozilla.com'],
+        depends_on_past=False,
+        dag=dag,
+    )
+    
+    
+    telemetry_derived__smoot_usage_new_profiles_compressed__v2 = bigquery_etl_query(
+        destination_table='smoot_usage_new_profiles_compressed',
+        dataset_id='telemetry_derived',
+        project_id='moz-fx-data-shared-prod',
+        owner='jklukas@mozilla.com',
+        email=['jklukas@mozilla.com'],
+        depends_on_past=False,
+        dag=dag,
+    )
+    
+    
+    wait_for_telemetry_derived__smoot_usage_new_profiles__v2 = ExternalTaskSensor(
+        task_id='wait_for_telemetry_derived__smoot_usage_new_profiles__v2',
+        external_dag_id='bqetl_kpi_dashboard',
+        external_task_id='telemetry_derived__smoot_usage_new_profiles__v2',
+        dag=dag,
+    )
```

Comment:
I'll add a check that if dependencies are defined in the same DAG, this is not necessary